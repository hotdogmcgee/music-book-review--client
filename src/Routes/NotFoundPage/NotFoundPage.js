import React from "react";
import { Section } from "../../Components/Utils/Utils";
import { Link } from "react-router-dom";
import "./NotFoundPage.css";

export default class NotFoundPage extends React.Component {
  render() {
    return (
      <Section className="NotFoundPage">
        <Link to="/">Music Book Review</Link>
        <h2>404 - Page not found</h2>
        <p>Try going back to your previous page.</p>
      </Section>
    );
  }
}
