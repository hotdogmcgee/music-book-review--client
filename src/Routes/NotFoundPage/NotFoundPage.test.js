import React from "react";
import NotFoundPage from "./NotFoundPage";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("NotFoundPage component", () => {
  it("renders a NotFoundPage by default", () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
