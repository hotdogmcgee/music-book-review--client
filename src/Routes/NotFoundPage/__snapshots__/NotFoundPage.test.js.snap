// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component renders a NotFoundPage by default 1`] = `
<Section
  className="NotFoundPage"
>
  <Link
    to="/"
  >
    Music Book Reviews
  </Link>
  <h2>
    404 - Page not found
  </h2>
  <p>
    Try going back to your previous page.
  </p>
</Section>
`;
